tests/src/JIT/MyRepro/test.cs

```diff
@@ -0,0 +1,34 @@
+using System;
```

Comment:
Can we do a bit better with the name f this test than "MyRepro"?

Reply:
@AaronRobinsonMSFT I'll delete this test when i'm done. This is just my lazy way of testing a simple pinvoke on the arm/arm64 cases with some form of gc stress :)